docs/ldml/tr35-info.md

```diff
@@ -1116,87 +1116,110 @@ The examples in #4 are due to the following ordering of the `unitQuantity` eleme
-<unitPreference regions="SE" skeleton="precision-increment/1">meter</unitPreference>
-```
-2. Meter is **metric**, mile-scandinavian is **metric_adjacent** so they both match the key-value ms-**metric**, so no change is made.
+#### Compute the category
 
-**Example B: xx-GB-u-ms-ussystem, volume, fluid**
-1. Fetch the data from `<unitPreferences category="volume" usage="fluid">` for xx-GB
-```
-<unitPreference regions="GB">gallon-imperial</unitPreference>
-<unitPreference regions="GB">fluid-ounce-imperial</unitPreference>
-```
-2. At least one of {gallon-imperial, fluid-ounce-imperial} does not match ms-**ussystem** so the locale is shifted to xx-**US**, and uses the following:
-```
-<unitPreference regions="US">gallon</unitPreference>
-<unitPreference regions="US">quart</unitPreference>
-<unitPreference regions="US">pint</unitPreference>
-<unitPreference regions="US">cup</unitPreference>
-<unitPreference regions="US">fluid-ounce</unitPreference>
-<unitPreference regions="US">tablespoon</unitPreference>
-<unitPreference regions="US">teaspoon</unitPreference>
-```
-
-APIs should clearly allow for both the use of unit preferences with the above process, and for the _invariant use_ of a unit measure.
-That is, while an application will usually want to obey the preferences for the locale or in the locale ID, there will definitely be instances where it will want to not use them.
-For example, in showing the weather, an application may want to show:
+A **category** is determined as follows from the input unit:
 
-High today: 68°F (20°C)
-
-To do that, the application needs to show the first value with the locale information, and then (a) query what the alternative is, and show the temperature in that.
-As an example, ICU only uses the unit preferences (with rg, ms, and/or mu and the likely region) in formatting units when a usage parameter is set.
+1. From the input unit, use the conversion data in [baseUnit](tr35-info.html#Unit_Conversion) and let the **input base unit** be the baseUnit attribute value.
+    * eg, for `pound-force` the baseUnit is `kilogram-meter-per-square-second`.
+2. If there is no such base unit (such as for a an unusual unit like `ampere-pound-per-foot-square-minute`), convert the input unit to a combination of base units, reduce to lowest terms, and normalize. Let the **input base unit** be that value.
+    * eg, `ampere-pound-per-foot-square-minute` -> `kilogram-ampere-per-meter-square-second`
+3. If the **input base unit** has a unitQuantity element, then let the **category** be the quantity attribute value.
+    * eg, `force` from `<unitQuantity baseUnit='kilogram-meter-per-square-second' quantity='force'/>`
+3. If the **input base unit** does not have a unitQuantity, let the output unit be the input base unit or an equivalent metric/SI unit, and return. This terminates the algorithm; there is no need to use the unit preferences information.
```

Comment:
I didn't understand this part ("an equivalent metric/SI unit"). How can we find the equivalent metric/SI unit?

We have a method to convert any unit to the base unit, but is there no way to convert to metric/SI units?

Reply:
What I didn't want to do is proscribe a particular method of producing a shorter metric unit.

Reply:
Actually, there is an example right below this. But I'll change the wording to be clearer.